modules/rhel-compute-available-hooks.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-rhel-compute.adoc
+
+[id="rhel-compute-available-hooks-{context}"]
+= Available hooks for RHEL compute machines
+
+You can use the following hooks when you update the Red Hat Enterprise Linux (RHEL)
+compute machines in your {product-title} cluster.
+
+
+[cols="1,1",options="header"]
+|===
+|Hook name |Description
+
+
+|`openshift_node_pre_cordon_hook`
+a|- Runs *before* each node is cordoned.
+- This hook runs against *each node* in serial.
+- If a task must run against a different host, the task must use
+link:http://docs.ansible.com/ansible/playbooks_delegation.html#delegation[`delegate_to` or `local_action`].
+
+|`openshift_node_pre_upgrade_hook`
+a|- Runs *after* each node is cordoned but *before* it is updated.
+- This hook runs against *each node* in serial.
+- If a task must run against a different host, the task must use
+link:http://docs.ansible.com/ansible/playbooks_delegation.html#delegation[`delegate_to` or `local_action`].
+
+|`openshift_node_upgrade_hook`
+a|- Runs *after* each node is updated but *before* it is uncordoned.
+- This hook runs against *each node* in serial.
+- If a task must run against a different host, they task must use
+link:http://docs.ansible.com/ansible/playbooks_delegation.html#delegation[`delegate_to` or `local_action`].
+
+|`openshift_node_upgrade_post_hook`
```

Comment:
should be
openshift_node_post_upgrade_hook

https://github.com/openshift/openshift-ansible/blob/master/HOOKS.md#openshift_node_post_upgrade_hook